docs/guides/concepts.md

```diff
@@ -55,7 +55,8 @@ rpc SayHello(HelloRequest) returns (HelloResponse){
 
 - Server streaming RPCs where the client sends a request to the server and gets
   a stream to read a sequence of messages back. The client reads from the
-  returned stream until there are no more messages.
+  returned stream until there are no more messages. gRPC guarantees message
+  ordering.
```

Comment:
nit: gRPC guarantees message order for a given stream.   The messages from different streams can be reordered.

Reply:
Away from a github enabled machine - would either of these wordings be preferred? 
A) "gRPC guarantees message ordering within an individual RPC call." 
B) "gRPC guarantees message ordering within an individual stream."

Reply:
Let's say call, since stream is a transport specific thing.